packaging/csapi-tizenfx.spec

```diff
@@ -12,8 +12,8 @@
 %define DOTNET_LIBRARY_PATH /usr/share/dotnet.tizen/lib
 %define DOTNET_NUGET_SOURCE /nuget
 
-%define TIZEN_NET_RUNTIME_IDENTIFIERS 10.0.0:9.0.0:8.0.0:7.0.0:6.5.0:6.0.0:5.5.0:5.0.0:4.0.0
-%define TIZEN_NET_TARGET_FRAMEWORK_MONIKERS net8.0-tizen10.0:net8.0-tizen:net8.0:net6.0-tizen9.0:net6.0-tizen8.0:net6.0-tizen:net6.0:tizen10.0:tizen90:tizen80:tizen70:tizen60:tizen50:tizen40
+%define TIZEN_NET_RUNTIME_IDENTIFIERS 11.0.0:10.0.0:9.0.0:8.0.0:7.0.0:6.5.0:6.0.0:5.5.0:5.0.0:4.0.0
+%define TIZEN_NET_TARGET_FRAMEWORK_MONIKERS net8.0-tizen11.0:net8.0-tizen10.0
```

Comment:
```suggestion
%define TIZEN_NET_TARGET_FRAMEWORK_MONIKERS net8.0-tizen11.0:net8.0-tizen10.0:net8.0-tizen:net8.0:net6.0-tizen9.0:net6.0-tizen8.0:net6.0-tizen:net6.0:tizen10.0:tizen90:tizen80:tizen70:tizen60:tizen50:tizen40
```

Reply:
Thanks for checking. I will apply the change.